Competition2017/src/competition/subsystems/agitator/commands/IntakeAgitatorCommand.java

```diff
@@ -0,0 +1,28 @@
+package competition.subsystems.agitator.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.agitator.AgitatorSubsystem;
+import competition.subsystems.agitator.commands.IntakeAgitatorCommand;
+import xbot.common.command.BaseCommand;
+
+public class IntakeAgitatorCommand extends BaseAgitatorCommand {
```

Comment:
We should use clearer verbiage; maybe `Feed`. I could see arguments for "intake" meaning both possible directions.